src/3box.js

```diff
@@ -17,7 +17,8 @@ const LevelStore = require('datastore-level')
 
 const ACCOUNT_TYPES = {
   ethereum: 'ethereum',
-  ethereumEOA: 'ethereum-eoa'
+  ethereumEOA: 'ethereum-eoa',
+  ethereumContract: 'ethereum-contract'
```

Comment:
👍 , or maybe this should be called `ERC1271`?

Reply:
I would stick to some generic naming, as there are no confirmed standard for this yet. We could extend the validation functions later, but the account type will still be the Smart Contract based accound

Reply:
The main reason I would prefer `ERC1271` here would be that if another standard emerges there would be an easy way to distinguish between links made in this way and links made with a new standard :)

Reply:
You're right!
How about putting that info into the `linkData`?

Reply:
This would end up in the `type` property as it is now right?

Reply:
:D right
Ok, so we can either create a sub-type, or just rename this type into smth else. You choose ;)
btw, in your spec `3ip-5` you wrote that the value should be `ethereum-contract`, so this spec should be changed as well

Reply:
Yeah, let's update it in the `3ip-5` spec and change this type :)